packages/dashboard/src/services/websocket.ts

```diff
@@ -0,0 +1,7 @@
+import { DerivWS } from 'Types';
+
+let ws: DerivWS = null;
+
+export const useWs = (): DerivWS => ws;
```

Comment:
IMO we should only use the `use` prefix for hooks 😅 
Maybe we can rename it to `getWs`?

Reply:
It is meant to be used as a hook.